src/components/AddFoodForm.js

```diff
@@ -0,0 +1,51 @@
+import { Divider, Input } from 'antd';
+import { useState } from 'react';
+
+// Iteration 4
+function AddFoodForm(props) {
+    const [name, setName] = useState("")
+    const [image, setImage] = useState("")
+    const [calories, setCalories] = useState("")
+    const [servings, setServings] = useState("")
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        const newFood = {
+            name: name,
+            image: image,
+            calories: calories,
+            servings: servings,
+        };
+
+        props.setFood((prevFoodArr) => {
+            return [newFood, ...prevFoodArr];
+        });
+
+        setName("");
+        setImage("");
+        setCalories("");
+        setServings("");
+    }
+    return (
+        <form onSubmit={handleSubmit}>
+            <Divider>Add Food Entry</Divider>
+
+            <label>Name</label>
+            <Input value={name} type="text" onChange={(e) => { setName(e.target.value) }} />
+
+            <label>Image</label>
+            <Input value={image} type="text" onChange={(e) => { setImage(e.target.value) }} />
+
+            <label>Calories</label>
+            <Input value={calories} type="text" onChange={(e) => { setCalories(e.target.value) }} />
```

Comment:
for Calories and Serving, you need to change the type to number. in this way you can prevent to getting other input(instead of number) from user and your code will not break ,